tests/main/interfaces-broadcom-asic-control/test-snapd-sh/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: test-snapd-sh
+summary: A no-strings-attached, no-fuss shell for writing tests
```

Comment:
The summary here (and everywhere) is no more accurate. I guess we can drop it entirely?

Reply:
We discussed this on standup and the agreement is to update all the names (including snap names) and summaries in a followup to reduce the diff (changing names would affect task.yamls).